pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pytransloadit"
-version = "0.2.2"
+version = "0.3.0"
```

I guess this needs to be 1.0 as well then
```suggestion
version = "1.0.0"
```